src/ResourceManager/Network/Commands.Network/help/Update-AzureRmVpnGateway.md

```diff
@@ -0,0 +1,258 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/update-azurermvpngateway
+schema: 2.0.0
+---
+
+# Update-AzureRmVpnGateway
+
+## SYNOPSIS
+Update-AzureRmVpnGateway updates a scalable VPN Gateway to the appropriate goal state.
+
+## SYNTAX
+
+### ByVpnGatewayName (Default)
+```
+Update-AzureRmVpnGateway -Name <String> -ResourceGroupName <String>
+ [-VpnConnection <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSVpnConnection]>]
```

Comment:
array not list

Reply:
Was already fixed in code. Regenerated help.